pkg/pillar/scripts/device-steps.sh

```diff
@@ -6,6 +6,7 @@
 USE_HW_WATCHDOG=1
 CONFIGDIR=/config
 PERSISTDIR=/persist
+PERSISTCONFIGDIR=/persist/config
 BINDIR=/opt/zededa/bin
```

Comment:
We need to start a new documentation under /doc documenting files that we put into /persist/config and /config and what is their purpose.

Reply:
@rvs can you file a separate issue on that?

Reply:
https://github.com/lf-edge/eve/issues/75

Reply:
opened a new issue